README.md

```diff
@@ -52,38 +49,33 @@ Once you've followed the general-purpose ["getting started"](http://codepush.too
 npm install --save react-native-code-push
 ```
 
-As with all other React Native plugins, the integration experience is different for iOS and Android, so perform the following setup steps depending on which platform(s) you are targetting.
-
-## iOS Setup
-
-Once you've acquired the CodePush plugin, you need to integrate it into the Xcode project of your React Native app. To do this, take the following steps:
+The React Native plugin for CodePush is configured to be linked with your project automatically by [RNPM](https://github.com/rnpm/rnpm) upon installation via an NPM postinstall hook. To verify that the linking has succeeded, you should see the following output in your terminal after installation:
+```
+rnpm info Linking react-native-code-push android dependency 
+rnpm info Android module react-native-code-push has been successfully linked 
+rnpm info Linking react-native-code-push ios dependency 
+rnpm info iOS module react-native-code-push has been successfully linked 
+```
 
-### Plugin Installation (iOS)
+Next, as with all other React Native plugins, the integration experience is different for iOS and Android, so perform the following setup steps depending on which platform(s) you are targetting.
 
-1. Open your app's Xcode project
-2. Find the `CodePush.xcodeproj` file within the `node_modules/react-native-code-push` directory, and drag it into the `Libraries` node in Xcode
+## iOS Setup
 
-    ![Add CodePush to project](https://cloud.githubusercontent.com/assets/516559/10322414/7688748e-6c32-11e5-83c1-00d3e6758df4.png)
+*NOTE: these instructions only apply to version 1.7.2-beta and above of the plugin. For setup instructions pertaining to older versions, please refer to [this version of the docs](https://github.com/Microsoft/react-native-code-push/blob/28c4f2dd03448c393445800a5e0d828b69a614e3/README.md).*
```

Comment:
Should we hyperlink directly to the iOS setup section in the readme to make this easier?